docs/api.md

```diff
@@ -117,6 +117,16 @@ An object of functions that change the model's state. These functions take the m
 }
 ```
 
+Reducers may also listen to actions from other models by listing the 'model name' + 'action name' as the key.
+
+```js
+{
+  reducers: {
+    'otherModel/actionName': (state, payload) => state + payload,
```

Comment:
Technically, isn't this the "action type"?

Reply:
Yes. In this case action types are all namespaced by the model name.

And you're right, it's not very clear. How should we phrase this to make it clearer?